src/MLJNaiveBayesInterface.jl

```diff
@@ -141,4 +141,125 @@ MMI.is_pure_julia(::Type{<:MultinomialNBClassifier}) = true
 MMI.input_scitype(::Type{<:MultinomialNBClassifier}) = Table(Count)
 MMI.target_scitype(::Type{<:MultinomialNBClassifier}) = AbstractVector{<:Finite}
 
+"""
+$(MMI.doc_header(GaussianNBClassifier))
+
+
+`GaussianNBClassifier`: Gaussian Naive Bayes classifier. Assumes variables have a
+multivariate normal distribution. Good for real-valued data.
```

Comment:
nit: the second part is kind of contained in the first. But I'd agree that being explicit is better. Maybe something like

```
This is a classifier that models the data in each class `k` as samples drawn from a multivariate Gaussian distribution with mean `\\mu_k` and covariance `\\Sigma_k`. This model therefore considers the data as real-valued. 
```

(note that the "considers as" is exactly what MLJ does when it takes the scitype to be continuous; you could have ordered data (1, 2, 1, 1) that is then coerced to continuous (1.0, 2.0, 1.0, 1.0)).

Reply:
> Cool thanks for this! I've added a few comments though a more serious review will likely come from the active maintainers.

Awesome! Thank you for your reviews 😄